src/components/Explore/Explore.tsx

```diff
@@ -152,11 +150,12 @@ const Explore: React.FunctionComponent<{
   const currentMetricName = getMetricName(currentMetric);
 
   const initialLocations = useMemo(
-    () => initialFipsList.map(findLocationForFips),
+    () => initialFipsList.map(fipsCode => regions.findByFipsCode(fipsCode)!),
     [initialFipsList],
   );
+
   const indigeneousPopulationsLocations = useMemo(
-    () => ['00001', '00002'].map(findLocationForFips),
+    () => ['00001', '00002'].map(fips => regions.findByFipsCode(fips)!),
```

Comment:
Unrelated, we probably don't need to use memo anymore since we have the regions by FIPS in a dictionary already

Reply:
Looks like this removing this instance of useMemo was necessary - may need a slightly larger pass at removing useMemo in general here